libs/dao-data/project.json

```diff
@@ -22,6 +22,14 @@
         "buildableProjectDepsInPackageJsonType": "dependencies"
       }
     },
+    "vite-build": {
+      "dependsOn": [
+        {
+          "target": "build",
+          "projects": "self"
+        }
+      ]
+    },
```

Comment:
not sure if this is needed - dao-data does not use vite

Reply:
yeah that doesn't seem necessary

Reply:
hmm weird, looks like we do need this?

got to be something we can do about it -- ill look into it

<img width="1458" alt="Screen Shot 2022-10-12 at 5 02 22 PM" src="https://user-images.githubusercontent.com/5998100/195392435-e5d1722b-5d2c-4baa-bb35-59a6595aeda0.png">

Reply:
That error basically means that command needs to exist for libraries that it depends on. That's why I added it here